docs/docs/workspaces-and-credits-faq/README.md

```diff
@@ -0,0 +1,47 @@
+# Workspaces, Credits, and Other New Features for Pipedream Orgs
+
+We're excited to announce a new set of foundational changes and features! In short — orgs are now [workspaces](/workspaces), invocations are now [credits](/pricing/#credits), and you have access to new features: [free source invocations](/pricing/#source-credit-usage), unlimited team members, and more.
+
+[[toc]]
+
+## Orgs are now workspaces
+
+We've renamed organizations to [workspaces](/workspaces), which is more consistent with the language other products use. Anywhere you used to see references to "organizations", you'll now see "workspaces".
```

Comment:
```suggestion
We've renamed organizations to [workspaces](/workspaces), which is more consistent with the terminology other products use. Anywhere you used to see references to "organizations", you'll now see "workspaces".
```

Reply:
Changed "language" to "terminology" to follow Google's Developer docs style guide (use precise and clear language).